crates/jstz_cli/Cargo.toml

```diff
@@ -15,60 +15,60 @@ default-run = "jstz"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-jstz_kernel.workspace = true
-jstz_crypto.workspace = true
-jstz_proto.workspace = true
-jstz_core.workspace = true
-jstz_api.workspace = true
-jstz_rollup.workspace = true
-jstz_node.workspace = true
-octez.workspace = true
-clap = { version = "^4.4", features = ["derive"] }
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-bs58 = "0.5"
-hex = "0.4"
-dirs = "3.0"
-nix = { version = "^0.27.1", features = ["process", "signal"] }
-http = "1.0.0"
-fs_extra = "1.2"
-anyhow = "1.0.82"
-daemonize = "0.5.0"
-tempfile = "3.10.0"
-signal-hook = "0.3.17"
-boa_engine = { version = "0.17.0", features = ["fuzz"] }
-rustyline = "14.0.0"
-tezos-smart-rollup.workspace = true
+ansi_term.workspace = true
+anyhow.workspace = true
+boa_engine.workspace = true
+boa_gc.workspace = true
+bs58.workspace = true
+clap.workspace = true 
+clap_complete.workspace = true
+console.workspace = true
+crossterm.workspace = true
+daemonize.workspace = true
+derive_more.workspace = true
+dialoguer.workspace = true
+dirs.workspace = true
+env_logger.workspace = true
+fs_extra.workspace = true
+futures-util.workspace = true
+futures.workspace = true
+hex.workspace = true
+http.workspace = true
+in-container.workspace = true
+indicatif.workspace = true
+jstz_api = { path = "../jstz_api" }
```

Comment:
Out of curiosity, why can't we refer to internal crates with `.workspace`?

Reply:
Since we rely on `.workspace` to differentiate between external dependencies and internal crates. 
We rely on this in #542 when building all the workspace dependencies (`cargoDeps`). If we don't have this distinction, then we'd essentially get cache misses each time, as the workspace dependencies would include all internal crates (which change in almost every PR)